.gitignore

```diff
@@ -54,3 +54,5 @@ cypress.env.json
 cypress/screenshots
 cypress/snapshots
 cypress/videos
+
```

Comment:
Unnecessary new line

Reply:
This file was autogenerated, do you want me to change it anyway?